fetch-quotes.test.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable indent */
+import fetchQuotes from './fetch-quotes';
+
+describe('fetchQuotes', () => {
+
+    it('fetches a single quote from the futurama api', () => {
```

Comment:
The test needs to use async/await in order for the result of the fetch function to resolve:
```js
it('fetches a single quote from the futurama api', async () => {
        const actual = await fetchQuotes();
[... and so on]
```
This will make the current test fail - `actual` will turn out undefined once the promise resolves.